ovn-nb.xml

```diff
@@ -3041,6 +3041,18 @@
           </p>
         </column>
 
+        <column name="options" key="netbios_name_server">
+          <p>
+            The DHCPv4 option code for this option is 44.
+          </p>
+        </column>
+
+        <column name="options" key="netbios_node_type">
```

Comment:
The patch LGTM.

Can you please move the documentation for this option  - netbios_node_type to the appropriate section i.e here - https://github.com/ovn-org/ovn/blob/master/ovn-nb.xml#L3115